requirements.txt

```diff
@@ -2,5 +2,6 @@ Pillow
 lxml
 cssselect
 jieba
-beautifulsoup
+beautifulsoup  # Only on python2
```

Comment:
This is not working in python 3.5.4 as it does not support beautifulsoup. Even after changing the requires.txt with beautidulsoup4 the install didnt work.  Attached error file 
[error_goose_extract.txt](https://github.com/grangier/python-goose/files/1853584/error_goose_extract.txt)